code/ATMOSPHERICS/components/unary/vent_scrubber.dm

```diff
@@ -2,6 +2,8 @@
 	icon = 'icons/atmos/vent_scrubber.dmi'
 	icon_state = "map_scrubber"
 
+	req_one_access_txt = "24;10"

```

Comment:
I have that there to make multitool menus not work for people without access.

To keep the greytide from fucking with scrubbers and vents in the hallways